binaries/workerserver/main.go

```diff
@@ -8,7 +8,8 @@ import (
 
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/scootdev/scoot/common/endpoints"
-	"github.com/scootdev/scoot/runner/execer/fake"
+	"github.com/scootdev/scoot/runner/execer/execers"
+	osexec "github.com/scootdev/scoot/runner/execer/os"
```

Comment:
In general, we *can't* have every implementation in any one package. And that's part of the point of golang: implementations don't have to be related to the interface.

But I think Caitie made the perfectly cromulent point that for many interfaces, there are simple impls that are not fake but want to live together. I think for these, it's good to lump them together.

I think execer/os is right on the line. But, e.g., I think a git-based implementation of snapshot is too complex, and should be in its own package.

WDYT?